Makefile

```diff
@@ -188,6 +188,15 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+# generate a new shrinkwrap
+shrinkwrap:
+	@rm -rf node_modules
+	@rm npm-shrinkwrap.json
+	@npm install --no-optional
+	@npm install --no-optional # remove this when this is fixed in npm 3
+	@npm install -g shonkwrap
```

Comment:
I would recommend changing the default install directory or changing folder permissions. You should be able to install global packages without sudo. 

nvm handles this properly by installing in : `~/.nvm/nodeversion/lib`

I can change this to not attempt to install